### ISSUE TITLE
Youtube playback error when attempting to block ad

### ISSUE THREAD

**Issue by j0zh94** [uBlockOrigin/uBlock-issues#925]
### Prerequisites

- [ x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [ x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [x ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x ] I am running the latest version of uBlock Origin
- [x ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

[Youtube produces a playback error when attempting to block the ad. Video will continue to play as normal after 10-30 seconds depending on ad length]

### A specific URL where the issue occurs

[https://www.youtube.com/watch?v=-wOdL7HFNOs]

### Steps to Reproduce

1. [Launch Google Chrome with Ublock Origin]
2. [Select ad supported Youtube video]
3. [Error will occur when intro ad is played]

### Expected behavior:

[no error on launch]

### Actual behavior:

[What actually happened]

### Your environment

* uBlock Origin version: uBlock Origin 1.24.4
* Browser Name and version: Version 80.0.3987.122 (Official Build) (64-bit)
* Operating System and version: Windows 10 Pro 1903

**Comment by uBlock-user** [comment]
Happening with AdGuard too - https://github.com/AdguardTeam/AdguardFilters/issues/50711#issue-573289985

**Comment by gorhill** [comment]
Such issues have to be deemed filter issues unless proven otherwise, and especially more so when other blockers also suffer same issue -- I see other blockers also affected, AdGuard, [AdBlock](https://www.reddit.com/r/Adblock/comments/fbbfpc/adblock_isnt_letting_videos_load/), [Adblock Plus](https://adblockplus.org/forum/viewtopic.php?f=10&t=77370), etc.

**Comment by Fizbin99** [comment]
> 
> 
> Such issues have to be deemed filter issues unless proven otherwise, and especially more so when other blockers also suffer same issue -- I see other blockers also affected, AdGuard, [AdBlock](https://www.reddit.com/r/Adblock/comments/fbbfpc/adblock_isnt_letting_videos_load/), [Adblock Plus](https://adblockplus.org/forum/viewtopic.php?f=10&t=77370), etc.
> 
> Actually, there are reports this is happening even without a blocker: https://www.reddit.com/r/youtube/comments/fcel86/bug_get_this_playback_error_with_a_random_code/.

I don't see any posts in that thread that says they are getting this error without an Ad Blocker.

The problem is specific (possible geographic) ads on YT are making their way past uBlock and other ad blockers. If you disable uBlock this error doesn't occur. If you're one of the lucky ones that isn't getting these specific ads, then the error also doesn't occur. 

I have two YT accounts. One gets these specific ads (and the error) and the other account does not. The group of ads that cause this error have a white screen with "LEARN MORE", "SHOP NOW", "TRY FREE" at the bottom.

Sometimes these ads will just show up (with uBlock activated) without the error. Here's an example...

![Try Free](https://user-images.githubusercontent.com/37309649/76038306-2396a200-5efe-11ea-9651-91d6c991f98a.png)

**Comment by uBlock-user** [comment]
Are you using NoScript with uBO ?

**Comment by Fizbin99** [comment]
Hi! No, Other than uBlock I only use Lastpass and Stylus.

**Comment by gorhill** [comment]
> I don't see any posts in that thread that says they are getting this error without an Ad Blocker.

See:
- https://www.reddit.com/r/youtube/comments/fdu491/bug_consistently_getting_this_bug_when_trying_to/fjl0rcy
- https://www.reddit.com/r/youtube/comments/fcel86/bug_get_this_playback_error_with_a_random_code/fjcf3io

**Comment by gwarser** [comment]
I don't know how much relevant this is, but one user [here](https://www.reddit.com/r/firefox/comments/fdkeou/why_does_this_happens_to_me_my_friends_dont_have/fjkwk3o/) report that YT is serving unskippable ads when ad blocker is disabled.

**Comment by ner00** [comment]
> > I don't see any posts in that thread that says they are getting this error without an Ad Blocker.
> 
> See:
> 
>     * https://www.reddit.com/r/youtube/comments/fdu491/bug_consistently_getting_this_bug_when_trying_to/fjl0rcy
> 
>     * https://www.reddit.com/r/youtube/comments/fcel86/bug_get_this_playback_error_with_a_random_code/fjcf3io

Quite a few comments from those links reinforce the idea that ad-blocking is the root cause. I can also confirm, the issue is related to ad-blocking. I was having this issue, infrequently, but happened every day at least once. Refreshing the page would never resolve during ~10sec (ad time), after which the problem would go away probably for hours if not for an entire day.

After uninstalling all ad-blockers I have not had the playback issue even once in 2 weeks, but I did have a lot of ads instead. Looking for reasons to disregard an obvious issue doesn't seem to be the correct approach, but maybe Google just check-mated ad-blockers somehow. In other places I have also seen people making comments about some accounts having more issues than others (or some not at all, depending on the ad targeting algorithm I assume), but it seems indisputable at this point that ad-blocking is the major, if not THE, trigger.

I have reinstalled the ad-blocker because it is still better than without it, for now at least because I assume it will get more aggressive from YT in the future.
